### ISSUE TITLE
Condition "show if is checked" is not working

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#716]
**Steps to reproduce**
1. Create a new form with an `Agree` field + a `Single-line Text` field
2. Add a condition on the `Single-line Text` field
3. Setup the condition to _I want to **Show** this field if **All** of the following rules match._: **Field:** the `Agree` field / **Condition:** `is` / **Value:** `Checked`
4. Try the form on frond-end, the `Single-line Text` field is never showing up
5. If you edit and reverse the condition to _I want to **hide** this field if **All** of the following rules match._: **Field:** the `Agree` field / **Condition:** `is not` / **Value:** `Checked` it works

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.5
- Craft version: 3.7.28
- Multi-site: Yes

**Comment by engram-design** [comment]
This was an unexpected reversion due to https://github.com/verbb/formie/commit/8da02151d63aa182b2e11ed78a3c49b6657fb004 which I _thought_ I'd tested thoroughly with Agree and Checkboxes fields. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.6](https://github.com/verbb/formie/releases/tag/1.5.6)
